pytest_splunk_addon/splunk.py

```diff
@@ -466,11 +481,19 @@ def splunk_hec_uri(request, splunk):
     splunk_session.headers = {
         "Authorization": f'Splunk {request.config.getoption("splunk_hec_token")}'
     }
-    uri = f'{request.config.getoption("splunk_hec_scheme")}://{splunk["host"]}:{splunk["port_hec"]}/services/collector'
+    uri = f'{request.config.getoption("splunk_hec_scheme")}://{splunk["forwarder_host"]}:{splunk["port_hec"]}/services/collector'
     LOGGER.info("Fetched splunk_hec_uri=%s", uri)
 
-    return splunk_session, uri
+    for _ in range(RESPONSIVE_SPLUNK_TIMEOUT):
```

Comment:
splunk_hec_uri method shoulnd't wait for Splunk instance.   the waiting should be happening before splunk fixture executes.

Reply:
Updated.